.env.development

```diff
@@ -1,4 +1,4 @@
-REACT_APP_API_URL="https://api.horse.link"
+REACT_APP_API_URL="http://localhost:3003"
```

Comment:
Did you mean to push this?

Reply:
I think thats fine for dev to be defaulted to local

Reply:
Ok cool